lib/ash_json_api/api/api.ex

```diff
@@ -28,6 +29,11 @@ defmodule AshJsonApi.Api do
         doc: "Whether or not create a /schema route that serves the JSON schema of your API",
         default: false
       ],
+      open_api: [
+        type: {:spark_behaviour, OpenApiSpex.OpenApi},
```

Comment:
Are there any dangers to this if OpenApiSpex is an optional dependency? From my brief testing it didn't break anything, but I'm not sure if there is code that is expecting to be able to introspect these behaviours?

Reply:
Should be fine :) If not, that's a bug we should figure out how to deal with elsewhere.